solutions/usaco-1038.mdx

```diff
@@ -2,15 +2,20 @@
 id: usaco-1038
 source: USACO Silver 2020 US Open
 title: Social Distancing
-author: Qi Wang (Java), Tanish Tyagi (C++)
+author: Qi Wang (Java), Tanish Tyagi (C++), Melody Yu (Video)
 ---
 
 [Official Analysis](http://www.usaco.org/current/data/sol_socdist_silver_open20.html)
 
+## Video Solution
+
+Note: Video solution might not be the same as written solution.
+<Youtube id="mt8QhnFOkr0" />
```

Comment:
maybe videos should also be put in `LanguageSection`s? what does everyone else think?

Reply:
The code is in C++ but the algorithm description (first half or so) applies to any language; I'd be okay with moving it if needed, though.

Reply:
I'm somewhat against this since I think it's unlikely people will make three videos for each problem; if there is multi language support it's probably just all in one video. So maybe it's sufficient to list out what language(s) the video uses?

Reply:
if the language isn't available, it defaults to the language that is available.

Reply:
yeah, which is why it would be fine to put the video in LanguageSection (since all users would still be able to see it)... but it also makes sense that there won't be that many videos for each problem and that each video can have multiple languages so it's not really necessary